client/src/pages/ActivityPage/ActivityPage.tsx

```diff
@@ -0,0 +1,51 @@
+import "./ActivityPage.css";
+
+function ActivityPage() {
+  return (
+    <section className="activity">
+      <h1>Vos activités</h1>
+
+      <section className="activitypublished-recipes">
+        <h2>Mes recettes publiées</h2>
+        <figure>
+          <img
+            src="src\assets\tests\Hachis Parmentier.jpg"
+            alt="Recette publiée"
+          />
+          <figcaption className="section-plus">
+            <p>Voir plus</p>
+          </figcaption>
+        </figure>
+      </section>
+
+      <section className="activitywait-recipes">
+        <h2>Mes recettes en attente</h2>
+        <section className="activity-recipes-list">
+          <figure className="waitin-img">
+            <img
+              className="img1"
```

Comment:
Ce genre de className n'est pas justifié, en ce que tu peux utiliser les sélecteurs du dessus.

`.activity-wait-recipes > section:first-of-type > figure` te permet d'aller modifier la balise figure directement.